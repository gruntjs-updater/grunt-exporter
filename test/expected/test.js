//<!--(start-js-export tmp/test.js)-->
alert("TEST");
//<!--(end-js-export)-->
