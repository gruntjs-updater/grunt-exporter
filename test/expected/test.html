/*
* Export from
* grunt-exporter
* https://www.npmjs.com/package/grunt-exporter
* https://github.com/stephanj79/grunt-exporter
*/

<!--(xxx includes/headerinner.html)-->